src/compas_model/scene/elementobject.py

```diff
@@ -54,7 +54,6 @@ class ElementObject(SceneObject):
 
     def __init__(
         self,
-        element: Element,
```

Comment:
accommodating the changes we made a while ago on `SceneObject`, where we make all inputs kwargs. In this case, element will be delivered by `item` inside kwargs.